components/result-table/result-table.jsx

```diff
@@ -4,14 +4,67 @@ import {
   Paper,
   Box
 } from "@material-ui/core";
+import { Table } from "react-bootstrap";
+import BootstrapTable from "react-bootstrap-table-next";
+import paginationFactory from "react-bootstrap-table2-paginator";
+import * as ReactBootStrap from "react-bootstrap";
```

Comment:
Using more libraries increases the size of the bundle and consequently the loading time of the website.
This might be a crazy concern from my part, bc users will be querying a lot of data anyways.

Do you think we could use the material-ui table? (bc it is already installed)

https://material-ui.com/components/tables/
This is something that occurred to me now. Idk what the implications would be.

Reply:
Sure, I will add that and see how it works.